batch/batch/worker/worker.py

```diff
@@ -84,6 +86,19 @@ def deeper_stack_level_warn(*args, **kwargs):
 
 warnings.warn = deeper_stack_level_warn
 
+
```

Comment:
Can you put a comment here explaining this patch, with a link to the broken line of code in aiodocker?